client/src/utils/Constants.ts

```diff
@@ -0,0 +1,3 @@
+export class Constants {
+    static infinity: number = Number.MAX_SAFE_INTEGER;
```

Comment:
You should probably use Typescript's `Infinity` directly. Test if it works and remove this file.

Reply:
Cool! I didn't even know this existed haha